lib/broadside/cli/gli_commands.rb

```diff
@@ -17,6 +17,11 @@ def add_instance_flag(cmd)
   cmd.flag [:n, :instance], type: Fixnum
 end
 
+def add_command_flags(cmd)
```

Comment:
FWIW i would agree with you except for these two facts:

1. we call these methods together no less than 4 times
2. we will _always_ need to call these two methods together if we ever add any future commands that run on instances.

4 is a lot of times - if it were only twice i would agree it was unnecessary but i think it serves to highlight what flags are required for commands run on instances.